docs/developers/sdk.md

```diff
@@ -237,103 +427,157 @@ At this point, the transactions is still in the coordinator's transaction pool a
 3. **fged** : Forged
 4. **invl** : Invalid
 
-. After a few seconds, we verify transaction status
+After a few seconds, we verify transaction status in the list of forged transactions to check if transaction has been processed:
 
 ```js
     // Get transaction confirmation
-    const txConf = await hermez.CoordinatorAPI.getHistoryTransaction(txInfo.id)
-    console.log(txConf)
+    const txXferConf = await hermez.CoordinatorAPI.getHistoryTransaction(txXferPool.id)
+    console.log(txXferConf)
 
 >>>>>
 
-{
-  L1orL2: 'L2',
-  id: '0x00000000000001e240004700',
-  itemId: 0,
-  type: 'Transfer',
-  position: 5,
-  fromAccountIndex: 'hez:DAI:4444',
-  toAccountIndex: 'hez:DAI:672',
-  amount: '4903020000000000000',
-  batchNum: 5432,
-  historicUSD: 49.7,
-  timestamp: '2019-08-24T14:15:22Z',
-  token: {
-    id: 4444,
-    ethereumAddress: '0xaa942cfcd25ad4d90a62358b0dd84f33b398262a',
-    name: 'Maker Dai',
-    symbol: 'DAI',
-    decimals: 18,
-    ethereumBlockNum: 539847538,
-    USD: 1.01,
-    fiatUpdate: null
-  },
-  L1Info: null,
-  L2Info: {
-    "fee": 36
-    "historicFeeUSD": 0.1
-    "nonce" : 121
+    {
+      L1Info: null,
+      L1orL2: 'L2',
+      L2Info: { fee: 1, historicFeeUSD: null, nonce: 1 },
+      amount: '2000',
+      batchNum: 15,
+      fromAccountIndex: 'hez:20_2:256',
+      fromBJJ: 'hez:1-WYg_cDxmLQPTxBDF2BdJYNsmK2KcaL6tcueTqWoQ6v',
+      fromHezEthereumAddress: 'hez:0x8401Eb5ff34cc943f096A32EF3d5113FEbE8D4Eb',
+      historicUSD: null,
+      id: '0x020000000001000000000000',
+      itemId: 9,
+      position: 0,
+      timestamp: '2020-12-17T21:41:53Z',
+      toAccountIndex: 'hez:20_2:257',
+      toBJJ: 'hez:_ayj1cwk6Kuch4oodEgYYTRWidBywlsV8cYlOyVPiZzl',
+      toHezEthereumAddress: 'hez:0x306469457266CBBe7c0505e8Aad358622235e768',
+      token: {
+        USD: null,
+        decimals: 18,
+        ethereumAddress: '0x2e9f55f7266d8c7e07d359daba0e743e331b7a1a',
+        ethereumBlockNum: 61,
+        fiatUpdate: null,
+        id: 3,
+        itemId: 4,
+        name: 'ERC20_2',
+        symbol: '20_2'
+      },
+      type: 'Transfer'
+    }
   }
-}
 ```
-
-
-### Withdrawing Funds from Hermez
-This is a 2-step process. First we need to do what we call an **Exit**. This moves the user's funds from their token account to a specific Exit merkle tree. Once the funds are in the Exit tree, we can then withdraw them to an Ethereum L1 account.
+If the API returns an object, it means that the transaction has been forged.
 
 ### Exit
```

Comment:
We should probably document Exit before Force Exit as it is the recommended mechanism

Reply:
This change is pending. IT will take a big longer

Reply:
What is pending? I just meant reordering the docs so this appears where Force Exit is, and move Force Exit to here

Reply:
well, the transaction details will be incorrect if i just reorder it.